cmake/Modules/FindDAGMC.cmake

```diff
@@ -0,0 +1,18 @@
+# Try to find DAGMC
+#
+# Once done this will define
+#
+#  DAGMC_FOUND - system has DAGMC
+#  DAGMC_INCLUDE_DIRS - the DAGMC include directory
+#  DAGMC_LIBRARIES - Link these to use DAGMC
+#  DAGMC_DEFINITIONS - Compiler switches required for using DAGMC
+
+find_path(DAGMC_CMAKE_CONFIG NAMES DAGMCConfig.cmake
+          HINTS ${DAGMC_ROOT}
```

Comment:
If you use a `DAGMC_ROOT` environment variable (as I tried to do), this doesn't work because it only looks for a cmake variable of that name. Can you add `$ENV{DAGMC_ROOT}` here too?

Reply:
Definitely. I'l make sure that gets propagated into main version of this file in the DAGMC repo too.